.gitlab-ci.yml

```
@@ -44,12 +45,20 @@ variables:
   - cargo +nightly --version
   - bash --version
 
+.check-passed-job:                 &check-passed-job
+  - if [ -e artifacts/cbs/passed-jobs/"$CI_JOB_NAME" ]; then exit 0; fi
+
+.pr-refs:                          &pr-refs
+  rules:
+    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+    - if: $CI_COMMIT_REF_NAME =~ /^cbs-.*/                          # Patched companion branches
+
 .common-refs:                      &common-refs
   # these jobs run always*
   rules:
     - if: $CI_PIPELINE_SOURCE == "schedule"
     - if: $CI_COMMIT_REF_NAME == "master"
-    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+    - !reference [.pr-refs, rules]
```

```suggestion
    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
    - if: $CI_COMMIT_REF_NAME =~ /^cbs-.*/                          # Patched companion branches
```

I would prefer not to mix styles

---

eb86bce408e6e793c80ff3e3d706b198b3dbc9e1